README.md

```diff
@@ -1,125 +1,27 @@
-# HospitalRun Frontend
+[Visit this page for general information on the HospitalRun application](https://github.com/HospitalRun/hospitalrun/blob/master/README.md) including:
 
-<div align="center">
```

Comment:
You need to restore this header with the badges

Reply:
✅